.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
In my view, such messages are inappropriate for users who do not use Nix.

```
cd pg_doorman
direnv: loading .envrc
direnv: using flake
environment:1320: nix: command not found
environment:1321: nix: command not found
```

Reply:
Fair point, I've updated `.envrc` so that it only uses Nix when it is available